lang/de.po

```diff
@@ -0,0 +1,40 @@
+msgid ""
+msgstr ""
+"Language: de\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Verwendung: vrms-rpm [Optionen]"
+msgid "msg_help_option_ascii"    
+msgstr "    Display rms ASCII-art wenn keine kommerziellen Pakete gefunden wurden\n"
```

Comment:
I'm not sure "kommerziellen" is the appropriate word to use here - there exists commercial free software. Wikipedia uses "proprietärer Software" / "freiheitsentziehender Software"; we should probably use one of these.